Packs/PrismaCloudCompute/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Prisma Cloud Compute",
     "description": "Use the Prisma Cloud Compute integration to fetch incidents from your Prisma Cloud Compute environment.",
     "support": "xsoar",
-    "currentVersion": "1.2.12",
+    "currentVersion": "1.2.13",
```

This integration already had an update to 1.2.13
Please change it to:

```suggestion
    "currentVersion": "1.2.14",
```

---

Done